src/kinto-build/README.md

```diff
@@ -9,25 +9,25 @@
 * Argo controller version `2.8.1` or higher
 
 ## Dependencies
-- [kinto-core](https://github.com/kintoproj/kinto-core)
+- [kinto-core](https://github.com/kintoproj/kintohub/tree/main/src/kinto-core)
```

Comment:
can use local link since it's in the same repo now?

Reply:
done